spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -258,8 +258,13 @@ def get_resolution_center(app_id, platform)
       parse_response(r, 'data')
     end
 
-    def get_rating_summary(app_id, platform, versionId = '')
-      r = request(:get, "ra/apps/#{app_id}/reviews/summary?platform=#{platform}&versionId=#{versionId}")
+    def get_rating_summary(app_id, platform, versionId = '', storefront = '')
```

Comment:
Should this be renamed to `get_ratings` because it is not longer return `AppRatingSummary` objects?